configs/components/pxp-agent.json

```diff
@@ -1 +1 @@
-{"location":"https://builds.delivery.puppetlabs.net/pxp-agent/202312203/artifacts/","version":"202312203"}
+{"location":"https://builds.delivery.puppetlabs.net/pxp-agent/202312203/artifacts/","version":"1.15.19"}
```

Comment:
I don’t think we want to make this change. 1.15.19 is the tag for the pxp-agent component contained in the pxp-agent-vanagon project. And the latter is a component in this vanagon repo